element_calcium_imaging/imaging.py

```diff
@@ -152,8 +151,23 @@ def make(self, key):
             else:
                 raise NotImplementedError('Unknown method: {}'.format(method))
         elif task_mode == 'trigger':
-            raise NotImplementedError(f'Automatic triggering of {method} analysis'
-                                      f' is not yet supported')
+            method = (ProcessingTask * ProcessingParamSet * ProcessingMethod * scan.Scan & key).fetch1('processing_method') 
+            if method == 'suite2p':
+                import suite2p
+
+                suite_ops = (ProcessingTask * ProcessingParamSet & key).fetch1('params')
+                suite_ops['save_path0'] = (ProcessingTask & key).fetch1('processing_output_dir')
```

Comment:
This alone is not sufficient I don't think. `processing_output_dir` is a relative path, so really you are passing into Suite2p a relative path, can it handle that?
```suggestion
                suite_ops['save_path0'] = find_full_path(get_imaging_root_data_dir(), (ProcessingTask & key).fetch1('processing_output_dir')).as_posix()
```

Reply:
The test notebook worked without crushing but I'm looking into this to see what's really going on.

Reply:
Yes this was a problem and it only works in one specific case. In the next PR it will be a full path.